Source/WebCore/PAL/pal/spi/cf/CFNetworkSPI.h

```diff
@@ -29,6 +29,7 @@
 
 #include <CFNetwork/CFNetwork.h>
 #include <pal/spi/cf/CFNetworkConnectionCacheSPI.h>
+#include <wtf/EnumTraits.h>
```

Comment:
We're trying to move to *.serialization.in instead of EnumTraits.

Reply:
Switched to use `*.serialization.in`.  Hope I did it correctly!